resources/views/link_stats.blade.php

```diff
@@ -55,7 +55,7 @@
 
     <div class="row bottom-padding">
         <div class="col-md-8">
-            <h4>Traffic over Time</h4> (total: {{ $link->clicks }})
+            <h4>Traffic over Time</h4> (total: {{ $link->clicks }}) (unique: {{ $uniques }})
```

Comment:
Placing the "unique" number of clicks next to the "total" number of clicks is a bit deceiving. The uniques count is calculated using the base rows from our existing stats, which only fetch rows from the given time period. If this is the behaviour we want, we should clarify this point, or move the count somewhere else.

Reply:
I've updated it to not take into account the date range.